src/tracker-component.js

```diff
@@ -35,6 +35,12 @@ type CartData = {|
     currencyCode? : string
 |};
 
+type CancelCartData = {|
+    cartId : string,
+    clientId : string,
+    encryptedAccountNumber : string
```

Comment:
why is encryptedAccountNumber required? Its a paypal understood attribute and merchants won't have this value...

Reply:
The EAN is not actually used in the delete function, which Bryan wrote. I'm not sure if he's using it for anything.